src/java.base/share/classes/java/lang/Process.java

```diff
@@ -589,4 +751,29 @@ public long skip(long n) throws IOException {
             return n - remaining;
         }
     }
+
+    /**
+     * A nested class to delay looking up the Charset for the native encoding.
+     */
+    private static class CharsetHolder {
+        private final static Charset nativeCharset;
+        static {
+            Charset cs;
+            try {
+                cs = Charset.forName(StaticProperty.nativeEncoding());
+            } catch (UnsupportedCharsetException uce) {
+                cs = Charset.defaultCharset();
+            }
+            nativeCharset = cs;
+        }
+
+        /**
+         * {@return Charset for the native encoding or {@link Charset#defaultCharset()}
```

Comment:
Need some edit here?

Reply:
Looks odd, but is the correct syntax for the new {@return javadoc tag}.
See https://bugs.openjdk.java.net/browse/JDK-8075778

Reply:
Hmm. Unless I'm missing something, you have mismatched curly braces.